src/decidim/electionguard/voter.py

```diff
@@ -22,8 +22,8 @@ def process_message(self, message_type: str, message: dict, context: Context):
 class ProcessJointElectionKey(ElectionStep):
     message_type = 'joint_election_key'
 
-    def process_message(self, message_type: str, message: dict, context: Context):
-        context.joint_key = deserialize_key(message['joint_election_key'])
+    def process_message(self, message_type: str, message: Content, context: Context):
+        context.joint_key = deserialize_key(message['content'])
```

Comment:
Shouldn't we access to the `joint_key` like in the [trustee](https://github.com/decidim/decidim-electionguard/pull/2/files#diff-926bc8b2cef6072e740ed575c0a51f86444e10932569505124d13a168dcabda4R135)?
```suggestion
        context.joint_key = deserialize_key(message['content']['joint_key'])
```